x-pack/legacy/plugins/ml/public/application/data_frame_analytics/pages/analytics_management/components/analytics_list/actions.tsx

```diff
@@ -26,10 +26,12 @@ export const AnalyticsViewAction = {
   render: (item: DataFrameAnalyticsListRow) => {
     const analysisType = getAnalysisType(item.config.analysis);
     const jobStatus = item.stats.state;
+    const isDisabled = isClassificationAnalysis(item.config.analysis);
```

Comment:
Should we disable the view button for all job types that aren't supported, for cases where the advanced editor has been used to create a type of job that isn't currently supported in the UI?

Reply:
Sounds like a good idea - from what I can see only `outlier detection`, `regression` and `classification` are the types of jobs so classification is the only one that we need to check for right now. 
Are you suggesting changing the check to be more like 
```
const isDisabled = !isRegressionAnalysis(item.config.analysis) && !isOutlierDetectionAnalysis(item.config.analysis);
```

Reply:
Updated in c09aa277087cc5f4466d8d2bd1ba19898b890800